### ISSUE TITLE
[Question] is this ready for production use?

### ISSUE THREAD

**Issue by simkimsia** [dabapps/django-readers#24]

**Comment by j4mie** [comment]
Good question! I'd say: if you're feeling adventurous 😉 

The existing API is simple enough that it's unlikely to change much (it may be added to). The specs are closely based on `django-rest-framework-serialization-spec`, which we do use in production (so we know the idea works), but this library itself hasn't been used in production yet. It will be soon, though, hopefully.

**Comment by simkimsia** [comment]
Do you (or dabapps) offer paid consultation for django developers? I feel I have more questions about this. So willing to pay for some support regarding this library. You can let me know how to reach out via email or you can DM me via twitter https://twitter.com/KimStacks

Thank you

**Comment by j4mie** [comment]
@simkimsia Yes, but I'm also more than happy to have a short chat without charging for it! I'll drop you a DM on Twitter soon.
